modules/tinymce/CHANGELOG.md

```diff
@@ -57,6 +57,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Moved the `print` plugin's functionality to TinyMCE core #TINY-8314
 - Renamed the `getShortEndedElements` Schema API to `getVoidElements` #TINY-8344
 - Changed the default statusbar element path delimiter from `»` to `›` #TINY-8372
+- Renamed `font_formats` option to `font_family_formats` #TINY-8328
+- Renamed `fontselect` toolbar button and `fontformats` menu item to `fontfamily` #TINY-8328
+- Renamed `formatselect` toolbar button and `blockformats` menu item to `blocks` #TINY-8328
+- Renamed `fontsize_formats` option to `font_size_formats` #TINY-8328
+- Renamed `fontsizeselect` toolbar button and `fontsizes` menu item to `fontsize` #TINY-8328
+- Renamed `lineheight_formats` option to `line_height_formats` #TINY-8328
+- Renamed `styleselect` toolbar button and `formats` menu item to `styles` #TINY-8328
```

Comment:
This has some grammar issues... Also I've moved some things around to keep similar items together.
```suggestion
- Renamed the `font_formats` option to `font_family_formats` #TINY-8328
- Renamed the `fontselect` toolbar button and `fontformats` menu item to `fontfamily` #TINY-8328
- Renamed the `fontsize_formats` option to `font_size_formats` #TINY-8328
- Renamed the `fontsizeselect` toolbar button and `fontsizes` menu item to `fontsize` #TINY-8328
- Renamed the `formatselect` toolbar button and `blockformats` menu item to `blocks` #TINY-8328
- Renamed the `styleselect` toolbar button and `formats` menu item to `styles` #TINY-8328
- Renamed the `lineheight_formats` option to `line_height_formats` #TINY-8328
```